### PR TITLE
Update ZipHandler and Methods

### PR DESCRIPTION
**Pull request by JeremyIR**
## Description

Updates ZipHandler with blob filepath support, adds MoveZip method for moving a file from 'unzip' into the specified subfolder, adds associated tests, and updates Docker infrastructure to properly navigate filesystem when moving .zips. 

Note: this PR contains a temporary fix for azure-function-core-tools, which tags it to a specific version. There is a [card in place to remediate](https://github.com/CDCgov/trusted-intermediary/issues/1581) this. 

## Issue

[1254](https://github.com/CDCgov/trusted-intermediary/issues/1254)

## Checklist

- [x] I have added tests to cover my changes
- [x] I have added logging where useful (with appropriate log level)

### CODE REVIEW

File: azure_functions/local.settings.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "AzureWebJobs.HttpExample.Disabled": "true",
     "AZURE_STORAGE_CONNECTION_STRING": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:12000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:12001/devstoreaccount1;",
     "POLLING_TRIGGER_QUEUE_NAME": "polling-trigger-queue",
+    "WEBSITE_TIME_ZONE": "America/Los_Angeles",
```
Comment (jcrichlake) [inline azure_functions/local.settings.json:9]: Remove this

File: docker-compose.yml [hunk 2]
```diff
@@ -73,7 +73,7 @@ services:
       - -c
       - |
         cd /home/site/wwwroot
-        npm install -g azure-functions-core-tools
+        npm install -g azure-functions-core-tools@4.0.6280
```
[the remark this answers is not in the document]
Comment (pluckyswan) [reply docker-compose.yml:76]: [Card](https://github.com/CDCgov/trusted-intermediary/issues/1581) created!

[the remark this answers is not in the document]
Comment (pluckyswan) [reply docker-compose.yml:76]: Should I link this PR to whose on call so they have a heads up?

[the remark this answers is not in the document]
Comment (pluckyswan) [reply docker-compose.yml:76]: I gave them a heads-up

Comment (somesylvie) [inline docker-compose.yml:76]: This change should probably be called out in the PR description

Comment (saquino0827) [reply docker-compose.yml:76]: I believe they made a follow up card to update this once the dependency is fixed

Comment (pluckyswan) [reply docker-compose.yml:76]: Good point, it was called out in a comment but that is less visible unless you hunt for it.

File: src/zip/zip.go [hunk 3]
```diff
@@ -74,17 +79,43 @@ func (zipHandler ZipHandler) Unzip(zipFilePath string) error {
...
 		return err
 	}
 
 	return nil
 }
 
+// MoveZip moves a file from 'unzip' into the specified subfolder e.g. 'success', 'failure'
+func (zipHandler ZipHandler) MoveZip(blobPath string, subfolder string) {
+	slog.Info("About to move file", slog.String("blobPath", blobPath), slog.String("destination subfolder", subfolder))
+	// url must include the container name e.g. 'sftp/unzip/cheeseburger.zip' vs 'unzip/cheeseburger.zip'
```
Comment (somesylvie) [inline src/zip/zip.go:108]: it might be helpful to denote somehow that the second example is a 'path' and doesn't need the container name

Comment (pluckyswan) [reply src/zip/zip.go]: I've updated this comment but am welcomed to better suggestions/rewriting

**Review by jcrichlake — APPROVED** [review]

**Pull request merged**